query.go

```diff
@@ -574,3 +577,140 @@ func aurInfoPrint(names []string) ([]*rpc.Pkg, error) {
 
 	return info, nil
 }
+
+func aurPkgbuilds(names []string) ([]string, error) {
+	pkgbuilds := make([]string, 0, len(names))
+	var mux sync.Mutex
+	var wg sync.WaitGroup
+	var errs multierror.MultiError
+
+	makeRequest := func(name string) {
+		defer wg.Done()
+
+		values := url.Values{}
+		values.Set("h", name)
+
+		url := "https://aur.archlinux.org/cgit/aur.git/plain/PKGBUILD?"
+
+		resp, err := http.Get(url + values.Encode())
```

Comment:
use https://golang.org/pkg/net/url/#Parse to make urls and join them

Reply:
why should I use that there? the values I encode there aren't a url of themselves and http.Get accepts a string and not a URL